src/main.cpp

```diff
@@ -2415,7 +2415,8 @@ void static InvalidBlockFound(CBlockIndex *pindex, const CValidationState &state
     if (state.IsInvalid(nDoS)) {
         std::map<uint256, NodeId>::iterator it = mapBlockSource.find(pindex->GetBlockHash());
         if (it != mapBlockSource.end() && State(it->second)) {
-            CBlockReject reject = {state.GetRejectCode(), state.GetRejectReason().substr(0, MAX_REJECT_MESSAGE_LENGTH), pindex->GetBlockHash()};
+            assert(state.GetRejectCode() < 0x100);
```

Comment:
Is this the "extra commit included by upstream in the merge outside the PR"?

Reply:
Yes. Upstream's GitHub shows it occuring in an unrelated PR, because it was not simply pushed to the original PR.